modules/dkan/dkan_datastore/dkan_datastore.pages.inc

```diff
@@ -122,8 +170,11 @@ function dkan_datastore_import_tab_form($form, &$form_state, $node) {
  * Submit function for import tab.
  */
 function dkan_datastore_import_tab_form_submit($form, &$form_state) {
+  module_load_include("php", "dkan_datastore", "src/Form/Import");
```

Comment:
Just wondering, but for everything noted as "dkan_datastore" - would this be potentially replaced with a plugin system and some configuration to define the selected data store?